ansible/molecule/default/tests/test_default.py

```diff
@@ -0,0 +1,27 @@
+import os
```

Comment:
The existing `konvoy-image-builder` is using `goss` packer plugin https://github.com/YaleUniversity/packer-provisioner-goss as a provisioning step that should test provisioned packages. Is this is a different level test? Should these 2 be unified?

Reply:
not sure if they should be unified. Theoretically they could be. 

With molecule you do not want to execute gpu tests if you work on something different and vice versa. Thats why we have two different scenarios.

I do not know goss it seems to be yaml based testing. I'm not sure how easy it is to adapt the same functionality. What was the reason to not use code based testing?

Reply:
goss is used to validate that a device is configured properly for a successful bootstrap. It is run after ansible provisioning to validate that the ansible was successful; but it is also capable of validating a device which was provisioned outside of KiB.

Reply:
for now I would not want to change anything. But my suggestion for the future would be that different molecule scenarios have different tests and the EE test merges them.

It doesn't really matter which technology is used.

With a second though I currently prefer the way its currently as we have two different tests ensuring things. The molecule tests are currently more for local development.